socrata-http-server-ext/src/main/scala/com/socrata/http/server/ext/StatusCode.scala

```diff
@@ -0,0 +1,8 @@
+package com.socrata.http.server.ext
+
+case class StatusCode(code: Int)
+
+object StatusCode {
+  val OK = StatusCode(200)
+  val NotFound = StatusCode(404)
```

Comment:
Just these two?

Reply:
Oh heh.  Completely forgot I'd only added the ones I needed.